core-addon/DataCollection.js

```diff
@@ -339,6 +338,10 @@ export default class DataCollection {
       processed["zipCode"] = data["zipcode"];
     }
 
+    if ("exactIncome" in data) {
+      processed['exactIncome'] = data["exactIncome"];
```

Comment:
Don't forget to update the schema for legacy telemetry :(

Reply:
indeed – will be filing a PR for this as well.